packages/mcp-server/src/tools/access-ip-address/create-access-ip-address.ts

```diff
@@ -0,0 +1,50 @@
+
+import { maybeFilter } from 'telnyx-mcp/filtering';
+import { Metadata, asTextContentResult } from 'telnyx-mcp/tools/types';
+
+import { Tool } from '@modelcontextprotocol/sdk/types.js';
+import Telnyx from 'telnyx';
+
+export const metadata: Metadata = {
+  resource: 'access_ip_address',
+  operation: 'write',
+  tags: [],
+  httpMethod: 'post',
+  httpPath: '/access_ip_address',
+  operationId: 'CreateAccessIpAddress',
+};
+
+export const tool: Tool = {
+  name: 'create_access_ip_address',
+  description:
+    "When using this tool, always use the `jq_filter` parameter to reduce the response size and improve performance.\n\nOnly omit if you're sure you don't need the data.\n\nCreate new Access IP Address\n\n# Response Schema\n```json\n{\n  $ref: '#/$defs/access_ip_address_response',\n  $defs: {\n    access_ip_address_response: {\n      type: 'object',\n      title: 'AccessIPAddressResponseSchema',\n      properties: {\n        id: {\n          type: 'string',\n          title: 'Id'\n        },\n        ip_address: {\n          type: 'string',\n          title: 'Ip Address'\n        },\n        source: {\n          type: 'string',\n          title: 'Source'\n        },\n        status: {\n          $ref: '#/$defs/cloudflare_sync_status'\n        },\n        user_id: {\n          type: 'string',\n          title: 'User Id'\n        },\n        created_at: {\n          type: 'string',\n          title: 'Created At',\n          format: 'date-time'\n        },\n        description: {\n          type: 'string',\n          title: 'Description'\n        },\n        updated_at: {\n          type: 'string',\n          title: 'Updated At',\n          format: 'date-time'\n        }\n      },\n      required: [        'id',\n        'ip_address',\n        'source',\n        'status',\n        'user_id'\n      ]\n    },\n    cloudflare_sync_status: {\n      type: 'string',\n      title: 'CloudflareSyncStatus',\n      description: 'An enumeration.',\n      enum: [        'pending',\n        'added'\n      ]\n    }\n  }\n}\n```",
+  inputSchema: {
+    type: 'object',
+    properties: {
+      ip_address: {
+        type: 'string',
+        title: 'Ip Address',
+      },
+      description: {
+        type: 'string',
+        title: 'Description',
+      },
+      jq_filter: {
+        type: 'string',
+        title: 'jq Filter',
+        description:
+          'A jq filter to apply to the response to include certain fields. Consult the output schema in the tool description to see the fields that are available.\n\nFor example: to include only the `name` field in every object of a results array, you can provide ".results[].name".\n\nFor more information, see the [jq documentation](https://jqlang.org/manual/).',
+      },
+    },
+    required: ['ip_address'],
+  },
+  annotations: {},
+};
+
+export const handler = async (client: Telnyx, args: Record<string, unknown> | undefined) => {
+  const { jq_filter, ...body } = args as any;
```

Comment:
there must be a way via stainless to avoid this force casting to `any` when handling args. I'm not sure how this will impact users consuming the library as there are no examples of projects consuming from 3.0.0-alpha yet but it's worth confirming if there's going to be type inference and auto complete when calling methods just cause handler args types here are not strict